firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/analytics/BlockingAnalyticsEventLogger.java

```diff
@@ -51,23 +51,25 @@ public BlockingAnalyticsEventLogger(
   @Override
   public void logEvent(@NonNull String name, @Nullable Bundle params) {
     synchronized (latchLock) {
-      Logger.getLogger().d("Logging Crashlytics event to Firebase");
+      Logger.getLogger()
```

Comment:
I think this line should remain DEBUG but keep the rest of the changes you made below about the analytics integration. If the message doesn't log successfully, we'll now see a warning or error. If there's nothing else printed to DEBUG, we can assume it succeeded.

Reply:
I'd intended this DEBUG log to cover that use case: https://github.com/firebase/firebase-android-sdk/pull/2362/files#diff-7db92a2657e4a466a2091ebc5233c28b7727d5a5c5121dd95d7e752518586256R783

The verbose log here just gives more details about any events logged through this class.

Thoughts?

Reply:
Ah yeah, looks good. 👍